notebook/static/tree/js/kernellist.js

```diff
@@ -85,7 +85,14 @@ define([
             .addClass('btn btn-warning btn-xs')
             .text(i18n._('Shutdown'))
             .click(function() {
-                var path = $(this).parent().parent().parent().data('path');
+                var parent = $(this).parent().parent().parent();
+                var path = parent.data('path'):
```

Comment:
This line should end in a semicolon.

Reply:
Thanks for adding it in to fix the build! I can't believe I sent that. 😊